### DISCUSSION TITLE
5b081a8b/

### DISCUSSION THREAD

**Discussion by giscus** [james1BadCreeper/james1BadCreeper.github.io#12] [Announcements]
# 5b081a8b/

天不生卷王，万古如长夜！
 - james1BadCreeper - james1 的博客

https://james1badcreeper.github.io/5b081a8b/

**Comment by Zimo-666** [comment]
$\Huge{James1badcreeper}\tiny{好厉害}$。

**Reply by james1BadCreeper** [reply]
Zimo 哥又大又强！
